src/domain/models/access-ticket.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseModel } from './base-model';
+import z from 'zod';
+
+export class AccessTicketModel extends BaseModel {
+  constructor(
+    private shlinkId: string,
+    private id?: string,
```

Comment:
Question: Is the `id` meant to be optional?

Reply:
The model is used for both creation and retrieval of the data from the databse. When we are creating an object in the db we do not require the Id hence optional.